docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
-will be excluded, which can also be useful for excluding things like blanks.
+.. figure::  images/generate_summary.png
+   :align:   center
+
+Refresh the page until the summary is displayed below. This can take up to 1 minute. You will now see a summary of this file displaying a histogram of the number of features per sample:
+
+.. figure::  images/histogram.png
+   :align:   center
+
+As you can see, this file contains 30 samples with between approximately 11,000 and 200,000 features, in our case, picked-OTUs (or observational taxanomic unit).
+
+Now we can begin analyzing these samples. Let’s go ahead and select “Process” which will take us to the commands selection page. Once there, select “dflt_name - BIOM” so that the commands pull down tab can be accessed which will initially display five actions. 
+
+.. figure::  images/command_options.png
+   :align:   center
+
+We will now go through the use of each command which will enable you to generate summaries, plot your data, calculate statistics to help you get the most out of your data. 
+
+Rarefying Data
+~~~~~~~~~~~~~~
+
+To start, the data must be rarefied. This means that all the samples in the analysis will be randomly subsampled to this number of features, in this case OTUs, reducing potential alpha and beta diversity biases. Samples with fewer than this number of features will be excluded, which can also be useful for excluding things like blanks. To choose a good cutoff for your data, view the histogram that was made when we generated the summary of the data.
+
+.. figure::  images/histogram.png
+   :align:   center
+
+An appropriate cutoff would exclude clear outliers, but retain most of the samples. Here we have already removed blanks from our data and eliminated the outliers prior to analysis so we will just use the minimum number of features observed in our samples (11030) as the cutoff.
+
+To rarefy the data, select "Rarefy" from the dropdown menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/rarify_parameter_without_number.png
+   :align:   center
+
+Several parameters will have only one option which will be automatically selected for you. In the "p-sampling-depth" field we will specify the number of features to rarefy our samples to. Enter "11030" in this box, and click "Add Command". 
+
+.. figure::  images/rarify_parameter_with_sampling_depth.png
+   :align:   center
 
-You can get a good idea of where to set this threshold by looking at the
-histogram generated by summarizing the input closed-reference OTU table, as
-discussed in :doc:`qiita-16S-analysis`. Here, it looks like 2100 would be an
-appropriate cutoff: it excludes one clear outlier, but retains most of the
-samples.
+Click the run button to start the process of rarefaction. The view will return to the original screen, while the rarefaction job runs. Refresh your browser every 10-20 seconds until the rarefied biom table appears: 
```

Comment:
Click the "Run" button above the workflow network to start the process of rarefaction. (...) click on the BIOM artifact to see blue "Jobs using this data" button. Once you click on it, you can see the current status of your job. You can also view it clicking on the burger in the top-right corner of the screen.  
Once the job is completed, you must refresh your browser window to see the result.